### PR TITLE
deprecated-tikv-importer-in-operator

### PR DESCRIPTION
**Pull request by Frank945946**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [ ] v1.5 (TiDB Operator 1.5 versions)
- [ ] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

**Review by Oreoxmt — APPROVED** [review]

File: en/restore-data-using-tidb-lightning.md [hunk 1]
```diff
@@ -8,16 +8,10 @@ aliases: ['/docs/tidb-in-kubernetes/dev/restore-data-using-tidb-lightning/']
 
 This document describes how to import data into a TiDB cluster on Kubernetes using [TiDB Lightning](https://docs.pingcap.com/tidb/stable/tidb-lightning-overview).
 
-TiDB Lightning contains two components: tidb-lightning and tikv-importer. In Kubernetes, the tikv-importer is inside the separate Helm chart of the TiDB cluster. And tikv-importer is deployed as a `StatefulSet` with `replicas=1` while tidb-lightning is in a separate Helm chart and deployed as a `Job`.
+In Kubernetes, the tidb-lightning is in a separate Helm chart and deployed as a `Job`.
 
 TiDB Lightning supports three backends: `Importer-backend`, `Local-backend`, and `TiDB-backend`. For the differences of these backends and how to choose backends, see [TiDB Lightning Backends](https://docs.pingcap.com/tidb/stable/tidb-lightning-backends).
 
-- For `Importer-backend`, both tikv-importer and tidb-lightning need to be deployed.
```
Comment (csuzhangxc) [inline en/restore-data-using-tidb-lightning.md:15]: update the above "TiDB Lightning supports three backends: `Importer-backend`," too?

File: en/restore-data-using-tidb-lightning.md [hunk 2]
```diff
@@ -8,16 +8,10 @@ aliases: ['/docs/tidb-in-kubernetes/dev/restore-data-using-tidb-lightning/']
 
 This document describes how to import data into a TiDB cluster on Kubernetes using [TiDB Lightning](https://docs.pingcap.com/tidb/stable/tidb-lightning-overview).
 
-TiDB Lightning contains two components: tidb-lightning and tikv-importer. In Kubernetes, the tikv-importer is inside the separate Helm chart of the TiDB cluster. And tikv-importer is deployed as a `StatefulSet` with `replicas=1` while tidb-lightning is in a separate Helm chart and deployed as a `Job`.
+In Kubernetes, the tidb-lightning is in a separate Helm chart and deployed as a `Job`.
 
 TiDB Lightning supports three backends: `Importer-backend`, `Local-backend`, and `TiDB-backend`. For the differences of these backends and how to choose backends, see [TiDB Lightning Backends](https://docs.pingcap.com/tidb/stable/tidb-lightning-backends).
```
Comment (Oreoxmt) [inline en/restore-data-using-tidb-lightning.md:13]:
```suggestion
TiDB Lightning supports two backends: `Local-backend`, and `TiDB-backend`. For the differences of these backends and how to choose backends, see [TiDB Lightning Backends](https://docs.pingcap.com/tidb/stable/tidb-lightning-backends).
```

File: zh/restore-data-using-tidb-lightning.md [hunk 3]
```diff
@@ -8,16 +8,10 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/restore-data-using-tidb-lightning/']
 
 本文介绍了如何使用 [TiDB Lightning](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-overview) 导入集群数据。
 
-TiDB Lightning 包含两个组件：tidb-lightning 和 tikv-importer。在 Kubernetes 上，tikv-importer 位于单独的 Helm chart 内，被部署为一个副本数为 1 (`replicas=1`) 的 `StatefulSet`；tidb-lightning 位于单独的 Helm chart 内，被部署为一个 `Job`。
+TiDB Lightning 位于单独的 Helm chart 内，被部署为一个 `Job`。
 
 目前，TiDB Lightning 支持三种后端：`Importer-backend`、`Local-backend` 、`TiDB-backend`。关于这三种后端的区别和选择，请参阅 [TiDB Lightning 文档](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-backends)。
```
Comment (Oreoxmt) [inline zh/restore-data-using-tidb-lightning.md:13]:
```suggestion
目前，TiDB Lightning 支持两种后端：`Local-backend` 和 `TiDB-backend`。关于这两种后端的区别和选择，请参阅 [TiDB Lightning 文档](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-backends)。
```

**Review by D3Hunter — APPROVED** [review]

**Review by lichunzhu — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Pull request merged**